styles.css

```diff
@@ -0,0 +1,378 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: rgb(0, 122, 184);
+}
+
+h4 a {
+  color: #fff;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+li {
+  list-style: none;
+  flex: 1;
+  padding: 0 5px;
+  display: flex;
+  align-items: center;
+  border-right: 1px solid #d1d1d1;
+  justify-content: center;
+}
+
+.item {
+  color: #eee;
+  font-family: 'Cabin', sans-serif;
+  font-size: 80%;
+  font-weight: 600;
+  text-shadow: 1px 1px #555;
+}
+
+input {
+  background-color: rgb(225, 225, 225, 0.1);
+  background-image: url(images/search-icon.png);
+  background-size: 17%;
+  background-position: 4% 80%;
+  background-repeat: no-repeat;
+  border: none;
+  border-radius: 30px;
+  box-shadow: inset 0 2px 1px #333;
+  width: 80%;
+  height: 20px;
+  padding: 4px 0 2px 7px;
+}
+
+input:focus,
+input:active {
+  background-image: none;
+  outline: none;
+}
+
+.upper {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(images/hero.jpg);
+  background-position: 10% 95%;
+  background-repeat: no-repeat;
+  height: 93vh;
+  width: 100%;
+}
+
+.menu {
+  width: 73%;
+  height: 7%;
+  min-height: 20px;
+  display: flex;
+  background: linear-gradient(180deg, rgb(116, 116, 116) 47%, rgb(87, 87, 87) 48%, rgba(137, 137, 137, 1) 120%);
+  border: 0.5px solid #666;
+  margin-top: 6%;
+  border-radius: 7px;
+  box-shadow: 0 2px 1px rgba(112, 112, 112, 0.4);
+}
+
+.menu a,
+.input-div {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-right: 0.8px solid #555;
+  box-shadow: inset 1px 0 #999;
+}
+
+.menu a:hover {
+  text-decoration: none;
+  background-color: rgba(5, 5, 5, 0.4);
+}
+
+.menu a:first-child div span i {
+  font-size: 23px;
+}
+
+.input-div {
+  flex: none;
+  width: 135px;
+  border-right: none;
+}
+
+h2,
+h3,
+h4 {
+  font-family: 'Roboto', sans-serif;
+  color: #fff;
+  text-align: center;
+}
+
+h2 {
+  font-weight: 100;
+  font-size: 60px;
+  margin-top: 35px;
+}
+
+h3 {
+  font-weight: 300;
+  font-size: 32px;
+  margin-top: 20px;
+  width: 65%;
+}
+
+h4 {
+  font-weight: 300;
+  font-size: 24px;
+  margin-top: 30px;
+  width: 42%;
+  display: inline-block;
+}
+
+.play-icon {
+  width: 31px;
+  height: 31px;
+  background-image: url(images/play-icon.png);
+  background-position: 76.8% 0;
+  float: right;
+}
+
+.bottom {
+  margin-top: 1px;
+  width: 100%;
+}
+
+.thumbs {
+  display: flex;
+  width: 100%;
+}
+
+.thumb {
+  width: 25vw;
```

Comment:
You still have the horizontal overflow, caused by the width of .thumbs, change the width to` width: 24.7vw;`